### PR TITLE
Bash file to run memory tests

### PR DESCRIPTION
**Pull request by sanskriti-s**

### CODE REVIEW

**Comment by sanskriti-s** [comment]
Testing results:
Machine with 4 lvl page tables
```
Expecting 4-level pagetables
```
Machine with 5 lvl page tables
```
Detected la57 cpu support
Detected 5lvl config
Expecting 5-level pagetables
```

File: run-tests [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+EXPECT_5LVL=0
+SUPPORT_NO5LVL=0
+SUPPORT_LA57=0
+SUPPORT_5LVL=0
+
```
Comment (joe-lawrence) [inline run-tests]: Suggestion: move the `SUPPORT_*` variables down under the "Detect configuration" comments.  While `EXPECT_5LVL` is a "global variable", the `SUPPORT_*` variables are probably going to be only local to the configuration detection code.

File: run-tests [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+if [[ $EXPECT_5LVL -eq 1 ]] ; then
+	echo "Expecting 5-level pagetables"
+else
+	echo "Expecting 4-level pagetables"
+fi
+
+
+#if [[ $SUPPORT_LA57 -eq 1 &&  $SUPPORT_NO5LVL -eq 0 && $SUPPORT_5LVL -eq 1]] ; then
+#	EXPECT_5LVL=1
```
Comment (joe-lawrence) [inline run-tests]: These commented lines may be removed

File: run-tests [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+EXPECT_5LVL=0
+SUPPORT_NO5LVL=0
+SUPPORT_LA57=0
+SUPPORT_5LVL=0
```
Comment (joe-lawrence) [inline run-tests]: Suggestion: rename `SUPPORT_5LVL` to `SUPPORT_KERNEL_5LVL` or `SUPPORT_CONFIG_5LVL` to indicate that its a kernel configuration option.

**Comment by joe-lawrence** [comment]
:+1:

**Pull request merged**